lcs/agents/macs/macs.py

```diff
@@ -279,9 +340,17 @@ def form_action_set(self, action: int) -> ClassifiersList:
         return ClassifiersList(*matching)
 
 
+ChildClassifier = NamedTuple('ChildClassifier',
+                             [
+                                 ('classifier', Classifier),
+                                 ('parent', Classifier)
+                             ])
+
+
 class LatentLearning:
     def __init__(self, cfg: Configuration):
         self.cfg = cfg
+        self.covering_session_count = 0
```

Comment:
I was used here at line 622: `covering_session = "%4i" % self.covering_session_count`.
I removed that in the latest commit since it was not very useful.